src/app/components/wallet/Balance.vue

```diff
@@ -24,133 +24,142 @@ div
 import Vue from 'vue'
 
 import {
-    radixTokenManager,
-    RadixTransactionBuilder,
-    RadixAccount,
-    RRI,
-    RadixIdentity,
+  radixTokenManager,
+  RadixTransactionBuilder,
+  RadixAccount,
+  RRI,
+  RadixIdentity
```

Comment:
I don't know if this is set in tsconfig, but I prefer trailing commas, they make editing and git merges easier https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
added rule in tslint